blocks/embed/embed.js

```diff
@@ -112,14 +146,14 @@ export default function decorate(block) {
     wrapper.innerHTML = '<div class="embed-placeholder-play"><button title="Play"></button></div>';
     wrapper.prepend(placeholder);
     wrapper.addEventListener('click', () => {
-      loadEmbed(block, link, true);
+      loadEmbed(block, link, false);
```

Comment:
@Lauraa1003 We can define autoplay as toggleable by the author eg

```
embed (autoplay)
--- 
link
```

So you can check for the className `autoplay`: 

`const autoplay = block.classList.contains('autoplay')`